contracts/Swaplace.sol

```diff
@@ -107,7 +109,7 @@ contract Swaplace is SwapFactory, ISwaplace, IERC165 {
 
     _swaps[swapId].config = 0;
 
-    emit SwapCanceled(swapId);
+    emit SwapCanceled(swapId, _swaps[swapId].owner);
```

Comment:
The msg.sender is the one that canceled the swap, so we should use `msg.sender` instead of reading from memory the swap owner.

Reply:
Fixed here: https://github.com/blockful-io/swaplace-contracts/pull/189